endpoint_product_catalog/models/endpoint_mixin.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class EndpointMixin(models.AbstractModel):
+    _inherit = "endpoint.mixin"
```

Comment:
TODO: I think it's better to move this to endpoint.endpoint model because is not relevant for inheriting models (Eg: edi.endpoint)

Reply:
done